scss/_patterns_side-navigation.scss

```diff
@@ -117,8 +117,9 @@
     width: 100%;
     z-index: 102;
 
-    .p-side-navigation:target &,
-    [class*='p-side-navigation--']:target &,
+    // :target can be used to control the sidenav with an anchor element, but this was removed in PR #5084.
+    // .p-side-navigation:target &,
+    // [class*='p-side-navigation--']:target &,
```

Comment:
I'd recommend to remove commented out code, changes and relevant PRs can be tracked using git history.

```suggestion
```

Reply:
I agree. There is no point of keeping commented code. There is a clear reason to remove that.